enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -626,91 +626,72 @@ OSSM 2.4.
 
 #### Gateway Topology
 
-Users have the option to deploy their Gateways using two distinct topologies: shared gateways or dedicated gateways.
-Each topology addresses specific customer requirements, comes with different security implications (see [Security Policy](#security-policy)),
-and will be supported in OpenShift.
+A Kubernetes Gateway resource represents a proxy server (e.g., [Envoy]) and
+manages its life-cycle. Once deployed, it is programmed with route resources
+(e.g., HTTPRoute, GRPCRoute – collectively, xRoutes) to handle application
+traffic.
 
-##### Shared Gateway Topology
+There are two deployment options:
+
+* Dedicated Gateways
+* Shared Gateways (not yet supported)
```

Comment:
We have to support Shared Gateways as a default.  Shared Gateways means that multiple listener hostnames can be configured on the Gateway.

Reply:
```suggestion
* Shared Gateways
```